aws/logs_monitoring/lambda_function.py

```diff
@@ -532,6 +550,15 @@ def generate_metadata(context):
         "forwarder_memorysize": context.memory_limit_in_mb,
         "forwarder_version": DD_FORWARDER_VERSION,
     }
+
+    DD_FORWARDER_TELEMETRY_TAGS.append(
```

Comment:
It's better to avoid piggybacking here. I would rather create a helper function `set_forwarder_telemetry_tags`, and call it from somewhere in the `parse()` function (because we need to know the `source`).